llvm/utils/repo/Makefile

```diff
@@ -114,26 +123,46 @@ libcxx.a: libcxxabi.a
 		-D CMAKE_TOOLCHAIN_FILE=$(TOOLCHAIN_FILE)                     \
 		-D CMAKE_CXX_COMPILER=$(LLVM_BUILD)/bin/clang++               \
 		-D CMAKE_C_COMPILER=$(LLVM_BUILD)/bin/clang                   \
+		-D utils_dir=$(LLVM_REPO)/llvm/utils/repo                     \
 		-D CMAKE_INSTALL_PREFIX=$(LLVM)                               \
 		-D musl_install=$(MUSL)                                       \
 		../libcxx &&                                                  \
 	make -j 8 &&                                                          \
 	make install
 
+$(LLVM)/lib/libunwind.db: libunwind.a
+	cp $(LLVM_REPO)/build_libunwind/libunwind.db $@
+
+$(LLVM)/lib/linux/libcompiler-rt.db: libcompiler-rt.a
+	cp $(LLVM_REPO)/build_libcompiler-rt/libcompiler-rt.db $@
+
+$(LLVM)/lib/libc++abi.db: libcxxabi.a
+	cp $(LLVM_REPO)/build_libcxxabi/libc++abi.db $@
+
+$(LLVM)/lib/libc++.db: libcxx.a
+	cp $(LLVM_REPO)/build_libcxx/libc++.db $@
+
+install-libs: libunwind.a \
+	libcompiler-rt.a \
+	libcxxabi.a \
+	libcxx.a
+
+install-repo-libs-dbs: $(LLVM)/lib/libunwind.db \
+	$(LLVM)/lib/linux/libcompiler-rt.db \
+	$(LLVM)/lib/libc++abi.db \
+	$(LLVM)/lib/libc++.db
+
 .PHONY: all
-all :
-	$(MAKE) libunwind.a \
-		libcompiler-rt.a \
-		libcxxabi.a \
-		libcxx.a
+all:

```

Comment:
Shouldn’t this be an “install” target?

Reply:
Fixed in the commit https://github.com/SNSystems/llvm-project-prepo/pull/155/commits/ce1d66d51a4d69b1046d9f2abdb013408110f63b.